ui/client/test/ProcessDefinitionUtils-test.js

```diff
@@ -0,0 +1,145 @@
+import * as ProcessDefinitionUtils from "../common/ProcessDefinitionUtils";
+
+describe("getNodesToAddInCategory", () => {
+  it("should return all nodes available in category", () => {
+    const nodesToAdd = ProcessDefinitionUtils.getNodesToAddInCategory(processDefinition, "Category1")
+
+    expect(nodesToAdd).toEqual(processDefinition.nodesToAdd)
+  })
+
+  it("should filter out unavailable nodes in category", () => {
+    const nodesToAdd = ProcessDefinitionUtils.getNodesToAddInCategory(processDefinition, "Category2")
+
+    expect(nodesToAdd).toEqual([
```

Comment:
can we make this assertion a bit more understandable? ;)

Reply:
Done.